client/src/pages/ChefContent.js

```diff
@@ -0,0 +1,114 @@
+import React from "react";
+import Button from "@material-ui/core/Button";
+import Typography from "@material-ui/core/Typography";
+import Grid from "@material-ui/core/Grid";
+import { Edit } from "@material-ui/icons";
+import Paper from "@material-ui/core/Paper";
+import Avatar from "@material-ui/core/Avatar";
+import Divider from "@material-ui/core/Divider";
+import { makeStyles } from "@material-ui/core/styles";
+import chefPic from "../assets/chefPic.png";
+import chefBackground from "../assets/chefBackground.png";
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    minHeight: theme.spacing(80),
+    backgroundColor: "#F8F8FF",
+    maxHeight: "600px",
+  },
+  sideMenu: {
+    backgroundColor: "#FFF",
+    width: "100%",
+    height: "100%",
```

Comment:
100vh would be a more reliable way to fill the page

Reply:
Done